src/pages/Home.js

```diff
@@ -188,7 +188,6 @@ class Home extends React.Component {
             <div className="clearfix mxn2 mb4 flex">
                 <div className="inline-block clearfix mx-auto">
                   <button className="left btn btn-outline  x-group-item rounded-left" value='states' onClick={this.handleMapTypeChange} >States</button>
```

Comment:
We should remove this as well. The buttons make sense together as a toggle, but not when there is only one.

![screen shot 2017-11-27 at 3 00 19 pm](https://user-images.githubusercontent.com/780941/33286588-d2aaef52-d383-11e7-8521-98e316a24852.png)